src/main/pages/user-management/resource-management.adoc

```diff
@@ -11,77 +11,82 @@ folder: user-management
 [id="overview"]
 == Overview
 
-Resource API is designed to help control resources that Che users consume. Che admins can set caps for free resources to each resource type, as well as for each particular account.
+The Resource API manages the resources that are utilized by Che users. Che administrators set limits on the amount of resources available for each user acount. 
 
-There are two kinds of accounts which may be used in Che:
+There are two kinds of accounts used in Che:
 
-* _personal_ - an account that belongs to a user. Only one user can be related to this type of account and consume resources provided to this account.
-* _organizational_ - account that belongs to an link:organizations.html[organization]. This type of account allows consuming related resources for each organization member. The organization can have sub-organizations and distribute resources between them.
+* _personal_ - This account belongs to a user. Only one user per account. 
+* _organizational_ - This account belongs to an link:organizations.html[organization]. This type of account allows each member of the organization to use resources. Resources are distributed between organizations and sub-organizations.
 
-Since workspaces are the critical for users in the development flow, resources are mostly related to workspaces and runtimes.
+The amount and usage refers to the amount and usage of resources for workspaces and runtimes.
 
-Multi-user Che supports four types of resources:
+Multi-user Che supports the following type of resources:
 
-* *RAM* - defines the amount of RAM which can be used by running workspaces at the same time;
-* *Timeout* - defines the period of time that is used to control idling of user workspaces;
-* *Runtime* - defines the amount of workspaces which user can run at the same time;
-* *Workspace* - defines the amount of workspaces which user can have at the same time.
+* *RAM* -  Amount of RAM that workspaces use.
+* *Timeout* - Period of time that workspaces are idle.
+* *Runtime* - Number of workspaces that users run at the same time;
```

Comment:
I'd include the "can run" that was in the original text here -- it removes ambiguity (also, I assume the semicolon was a typo): "Number of workspaces that users can run at the same time."